data/fixtures/recorded/actions/followSplitVest.yml

```diff
@@ -0,0 +1,40 @@
+languageId: typescript
+command:
+  version: 7
+  spokenForm: follow split urn
+  action:
+    name: executeCommand
+    commandId: editor.action.revealDefinitionAside
```

Comment:
This doesn't test the new implementation

Reply:
Shoot. Apparently it was picking up my custom action instead. I'm having some trouble getting it to recognize this new action now. Do you have any tips to debug if it's not seeming to recognize the action at all?

Reply:
I forgot that I needed to symlink the cursor list settings in place of my own settings. Updated with the new test case.